<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>polyCountBudgetMainWindow</class>
 <widget class="QMainWindow" name="polyCountBudgetMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Your Polycount Budget</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="resetAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="budgetDock">
   <property name="windowTitle">
    <string>Poly Count Budget</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="polyCountTab">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="setBudgetTab">
        <attribute name="title">
         <string>Set Budget</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableView" name="setBudgetTableView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="polycontInScene">
        <attribute name="title">
         <string>Polycont in Scene</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTreeView" name="polycountTreeView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="resetAction">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
